app/router/group.py

```diff
@@ -31,12 +31,11 @@ def get_group_data(request: Request):
     """
     query_params = {}
     for key in request.query_params.keys():
-        if key not in ["name", "id"]:
+        if key not in ["name", "id", "input_schema", "locale", "locale_data"]:
```

Comment:
I don’t think the extra parameters are needed.

Reply:
removed unwanted params